go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/princjef/gomarkdoc
+module github.com/cloudogu/gomarkdoc
 
 go 1.18
```

Comment:
When `/go/doc/comment` is used and only provided starting from Go 1.19, why is the module set to 1.18?